src/main/java/com/exasol/repository/GitRepository.java

```diff
@@ -27,5 +27,6 @@
      * @param branchName name of a branch to get content from
      * @return new instance of {@link GitBranchContent}
      */
+    // [impl->dsn~gr-retrieves-branch-content~1]
     public GitBranchContent getRepositoryContent(String branchName);
```

Comment:
```suggestion
    public GitBranchContent getRepositoryContent(final String branchName);
```
Check if this is missing elsewhere too, please.

Reply:
I think `SaveActions` plugin doesn't add final modifiers in interfaces.. :( I had to add them manually.